test/verify/check-packagekit

```diff
@@ -1146,6 +1146,27 @@ ExecStart=/usr/local/bin/{packageName}
         # TODO make Packagekit GetUpdates work for tests properly
         b.wait_not_present("#available-updates")
 
+    @skipDistroPackage()
+    @nondestructive
+    def testCondition(self):
+        if os.environ.get("TEST_SCENARIO") != "pybridge":
+            self.skipTest("conditions not implemented on C bridge yet")
+
+        m = self.machine
+        b = self.browser
+
+        self.assertIn("\nupdates", m.execute("cockpit-bridge --packages"))
+
+        # pretend that this is an OSTree image, which fails path-not-exists: /sysroot/ostree
+        self.write_file("/sysroot/ostree", "")
+        self.addCleanup(m.execute, "rm -rf /sysroot")
```

Comment:
actually, make this `rmdir || true` and run before the write_file(). The latter cleans up /sysroot/ostree, but not the /sysroot directory itself.